Jenkinsfile

```diff
@@ -866,8 +866,9 @@ pipeline {
                                       rm -rf "$STAGE_NAME/"
                                       mkdir "$STAGE_NAME/"
                                       mv install/Linux/TESTING/avocado/job-results/CART_1node/* \
-                                         install/Linux/TESTING/testLogs-1_node \
-                                         "$STAGE_NAME/"
+                                         "$STAGE_NAME/" || true
```

Comment:
What happens if this fails for a reason other than the source doesn't exist?

Reply:
You mean if there are tests related failures in production Jenkins? Wouldn't Jenkins report that in the test result report and the overall status?